packages/app-desktop/gui/NoteEditor/NoteBody/CodeMirror/CodeMirror.tsx

```diff
@@ -731,15 +731,18 @@ function CodeMirror(props: NoteBodyEditorProps, ref: any) {
 	// It might be buggy, refer to the below issue
 	// https://github.com/laurent22/joplin/pull/3974#issuecomment-718936703
 	useEffect(() => {
-		function pointerInsideEditor(x: number, y: number) {
+		function pointerInsideEditor(params: any) {
+			const x = params.x, y = params.y, isEditable = params.isEditable, inputFieldType = params.inputFieldType;
 			const elements = document.getElementsByClassName('codeMirrorEditor');
-			if (!elements.length) return null;
+
+			// inputFieldType: To avoid context-menu flickering when there is an input field above the codeMirrorEditor.
```

Comment:
That's still no good I'm afraid and I feel this pull request is dragging on. In your comment you've explained why you need this parameter, I quote you:

> Because the input field type of CodeMirror is "textarea" so the inputFieldType = "none",
and any single-line input above codeMirror has inputFieldType value according to the type of input e.g.(text = plainText, password = password, ...).

That's a good explanation, so could you put it there please?

Reply:
Done, I don't know if there is another better solution but
I tested the code in different scenarios and it works fine.